files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -247,6 +250,14 @@
     - { key: "AllowTcpForwarding", value: "no" }
     - { key: "X11Forwarding", value: "no" }
     - { key: "HostKeyAlgorithms", value: "-ssh-rsa*" }
+    - { key: "MaxAuthTries", value: "6" }
+    - { key: "PermitRootLogin", value: "no"}
+    - { key: "Ciphers", value: "chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com"}
+    - { key: "MACs", value: "umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512"}
```

Comment:
This conflicts with a different setting for `MACs` earlier in this file. If we're in agreement that we should use an explicit list of MACs (and not an exclusionary list like before), then the other code should be deleted.